boards/shields/x_nucleo_nfc04a1/x_nucleo_nfc04a1.overlay

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		eeprom-0 = &eeprom0;
+		shield-green-led1 = &shield_green_led1;
+		shield-blue-led2 = &shield_blue_led2;
+		shield-yellow-led3 = &shield_yellow_led3;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		shield_green_led1: shield_green_led_1 {
+			gpios = <&gpiob 4 GPIO_ACTIVE_HIGH>;
+			label = "Shield LED1";
+		};
+		shield_blue_led2: shield_blue_led_2 {
+			gpios = <&gpiob 5 GPIO_ACTIVE_HIGH>;
+			label = "Shield LED2";
+		};
+		shield_yellow_led3: shield_yellow_led_3 {
+			gpios = <&gpioa 10 GPIO_ACTIVE_HIGH>;
+			label = "Shield LED3";
+		};
+	};
+};
+
+&usart1 {
+	pinctrl-0 = <&usart1_tx_pa9 &usart1_rx_pa10>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&arduino_i2c {
+	status="okay";
+
+	eeprom0: eeprom@53 {
```

Comment:
Please read https://docs.zephyrproject.org/latest/hardware/porting/shields.html#shield-porting-and-configuration